server/src/main/java/org/opensearch/cluster/coordination/UnsafeBootstrapClusterManagerCommand.java

```diff
@@ -81,7 +83,11 @@ public class UnsafeBootstrapClusterManagerCommand extends OpenSearchNodeCommand
     static final Setting<String> UNSAFE_BOOTSTRAP = ClusterService.USER_DEFINED_METADATA.getConcreteSetting(
         "cluster.metadata.unsafe-bootstrap"
     );
-
+    static final String REMOTE_CLUSTER_STATE_ENABLED_NODE =
+        "Unsafe bootstrap cannot be performed when remote cluster state is enabled as cluster state in the remote is considered the source of truth. "
```

Comment:
Unsafe bootstrap cannot be performed when remote cluster state is enabled. The cluster state in the remote store is considered the source of truth.